README.md

```diff
@@ -101,35 +103,16 @@ Across the board, we have some minor differences from the
 
 Other specific deltas will be noted in the code for the given metric.
 
-## Metric Prioritization
-
-### Table stakes quality:
-- `q_term_use` complies with US Core v1
-- `q_ref_target_pop` complies with US Core v1 (can be run on partial extracts)
-- `q_ref_target_valid` complies with US Core v1 (only on full extracts or data lake)
-- `q_valid_us_core_v4`
-  - numerator: resources that don't have all mandatory bits of any profile
-
-### Table stakes characterization:
-- `c_resource_count` (by category, year, month)
-- `c_pt_count` (by birth year gender, ethnicity, race)
-- `c_pt_deceased_count` (by gender, by age at death)
-- `c_term_coverage` (by resource type, by category)
-- `c_resources_per_pt` (include combinations?)
-- `c_us_core_v4_count`
-  - Tells how many rows match mandatory US Core support
-  - And for each separate must-support requirement, tells which rows have the value
-
-### High value quality:
-- `q_date_sequence`
-- `q_date_in_lifetime`
-- `q_date_recent`
-
-### High value characterization:
-- `c_element_use` for USCDI v1 “must support” elements
-- `c_date_precision` (by resource type, by category, by date element, by precision level)
-- `c_identifier_coverage` (by resource type)
-
-### Useful quality:
-- `q_obs_value_range`
-- `q_obs_comp_value_range`
+## Implemented Metrics
+
+- c_pt_count
+- c_pt_deceased_count
+- c_resource_count
+- c_resources_per_pt
+- c_term_coverage
+- c_us_core_v4_count
+- q_date_recent
+- q_ref_target_pop
+- q_ref_target_valid
+- q_term_use
+- q_valid_us_core_v4
```

Comment:
should these be links to the various readmes? or should there be a one-sentence description of what these things are about?

Reply:
The top bit of the readme links to the qualifier docs, where all these are defined. I don't want to link directly to these specific definitions yet, because they currently have really gross html anchors.

I'm in the middle of a big qualifier-side doc cleanup - I think once that's done it's a good idea to come back to this side and link to them individually, yeah.